src/nodes/tsl/TSLCore.js

```diff
@@ -553,6 +553,13 @@ export const Fn = ( jsFunc, nodeType ) => {
 
 };
 
+/**
+ * @function
+ * @deprecated since r168. Use {@link Fn} instead.
+ *
+ * @param  {...any} params
+ * @returns {Function}
+ */
```

Comment:
Do you mind adding `/** @module TSLCore **/` at the top of the module? Otherwise this function ends up in global scope.

Reply:
Never mind. Fixed it by myself.